vfile-reporter-folder-json.js

```diff
@@ -1,9 +1,10 @@
 const { sep } = require("path");
 
 class vFolder {
-  constructor({ name = "", children = [] } = {}) {
+  constructor({ name = "", children = [], parentPath = "" } = {}) {
     return {
       type: "folder",
+      path: parentPath + name + "/",
```

Comment:
consider using `path.resolve` or `path.join` to ensure cross platform support on windows.